### ISSUE TITLE
Can't use package constants when defining integer range types

### ISSUE THREAD

**Issue by John-Titor** [bagel99/esl#5]
```
// we can declare a range type using constants in the same namespace
const one = 1; const ten = 10;
type range_ok: one..ten;

// but we can't do it with constants out of a package
package limits { const rstart = 1; const rend = 10; }
var start: _int = limits.rstart;               // works as an initializer
type range_fail: limits.rstart..limits.rend;   // ... but not as range anchors
```
gives:
```
# ../src/eslc2 -L9.0.0 -m x86-64 01_range_from_package.esl > out.ll
./01_range_from_package.esl:8: Error: Undefined identifier: rstart
```

**Comment by bagel99** [comment]
Yep, that is a real bug.

**Comment by bagel99** [comment]
That's a real ugly bug.  I'm working on a solution, but it isn't pretty.

**Comment by bagel99** [comment]
Fixed in commit 1a53f6c5568510630c47c4b1b93bee27a8d88679
